west.yml

```diff
@@ -102,7 +102,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 83e5a13640417a2efb26a31f88b232e4dac31fb2
+      revision: pull/56/head
```

Comment:
The hal PR has now been merged, please update this to the new SHA

Reply:
Done, thank you